ckanext/depositar_theme/templates/home/index/idx_stats.html

```diff
@@ -0,0 +1,40 @@
+{% set formatCntList = h.get_format_count() %}
+
+<div class="idx stats">
+  <div class="container">
+    <div class="stat-data">
+      <div class="imp-data">
+        <div class="imp-item">
+          <h1>{{ _('Datasets') }}</h1>
+          <a href="{{ h.url_for('/dataset') }}" class="btn">
```

Comment:
改為 Flask endpoint (route) name (其他 `h.url_for` 請一併調整)

```suggestion
          <a href="{{ h.url_for('dataset.search') }}" class="btn">
```